cranelift/codegen/src/isa/aarch64/lower/isle.rs

```diff
@@ -227,4 +227,35 @@ where
     fn emit(&mut self, inst: &MInst) -> Unit {
         self.emitted_insts.push(inst.clone());
     }
+
+    fn cond_br_zero(&mut self, reg: Reg) -> CondBrKind {
+        CondBrKind::Zero(reg)
+    }
+
+    fn cond_br_cond(&mut self, cond: &Cond) -> CondBrKind {
+        CondBrKind::Cond(*cond)
+    }
```

Comment:
Should `CondBrKind` be moved into ISLE? This seems like a lot of code to write to glue the non-ISLE definition in with the ISLE code. Doesn't need to happen in this PR (or at all) unless you want, but just something I though of when looking at this code.

Reply:
Currently `CondBrKind` is pretty heavily used it looks like in the backend and since ISLE can only generate named-field enums instead of tuple-enums like `CondBrKind` is today I think I'll leave it defined externally, but I agree this'd be a good future cleanup!